### PR TITLE
Errorbages

### PR DESCRIPTION
**Pull request by MahmoudMH**
### Fixes
fixes #<issue_number>

### Related
related to #<issue_number>

### Summary
A couple of sentences explaining the contents of the PR

### Major Changes
[**OPTIONAL for small PRs**] Detailed description of major changes in the PR, ideally with references to relevant commits, issues, line-numbers, etc.

### CODE REVIEW

File: src/controllers/error.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+exports.client = (req, res) => {
+  res.render('404',{layout:false});
```
Comment (Ghadeer93) [inline src/controllers/error.js:2]: ok its good :+1:  , but now if i im 404 page how i can return back to home or anther page ? :relaxed: 
so i think you need to add button or any thing to do its :smile:

File: src/controllers/error.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+};
+exports.server = (err, req, res, next) => {
+  res.render('500',{layout:false});
```
Comment (Ghadeer93) [inline src/controllers/error.js]: i think need to change :wink:

**Review by Ghadeer93 — CHANGES_REQUESTED** [review]
need to change

File: src/controllers/error.js (hunk 1 above)
Comment (MahmoudMH) [reply src/controllers/error.js:2]: i did it ^_^ "

**Review by Ghadeer93 — APPROVED** [review]

**Pull request merged**